trail/upload/models.py

```diff
@@ -3,29 +3,31 @@
 """Define the SQL metadata design here."""
 
 
-class ExampleFrame(models.Model):
-    # meta-class mixin ??
-    class Meta:
-        # Django doesn't do composite primary keys
-        constraints = [
-            models.UniqueConstraint(fields=['run', 'camcol', 'filter', 'field'],
-                                    name="SDSS unique identifiers.")
-        ]
-
-    run = models.IntegerField()
-    camcol = models.IntegerField()
-    filter = models.CharField(max_length=1)
-    field = models.IntegerField()
-
-    ctype = models.CharField(max_length=20, null=False)
-    crpix1 = models.FloatField(null=False)
-    crpix2 = models.FloatField(null=False)
-    crval1 = models.FloatField(null=False)
-    crval2 = models.FloatField(null=False)
-
-    cd11 = models.FloatField(null=False)
-    cd12 = models.FloatField(null=False)
-    cd21 = models.FloatField(null=False)
-    cd22 = models.FloatField(null=False)
-
-    t = models.DateTimeField(verbose_name="UTC time at exposure start.")
+class Frame(models.Model):
+    # integer autoincrement primary fields is automatically added by Django
+
+    # verbose_names should be lowercase, Django will capitalize
+    # https://docs.djangoproject.com/en/3.1/topics/db/models/#verbose-field-names
+    metadata_translator_name = models.CharField("name of used translator.", max_length=10)
+    instrument = models.CharField("instrument name", max_length=20)
+    telescope = models.CharField("telescope", max_length=20, null=False)
+    science_program = models.CharField("science program image is a part of.", max_length=30)
+
+    obs_lon = models.FloatField("observatory longitude (deg)", null=False)
+    obs_lat = models.FloatField("observatory latitude (deg)", null=False)
+    obs_height = models.FloatField("observatory height (m)", null=False)
+
+    datetime_begin = models.DateTimeField("UTC at exposure start.", null=False)
+    datetime_end = models.DateTimeField("UTC at exposure end.", null=False)
+    exposure_duration = models.FloatField("exposure time (s)")
+    physical_filter = models.CharField("physical filter", max_length=30)
```

Comment:
Consider renaming to `filter_name` since this isn't specifically the physical piece of glass a la Rubin Science Pipelines.